demonstrations/tutorial_shadow_hamiltonian_simulation.py

```diff
@@ -0,0 +1,359 @@
+r"""Shadow Hamiltonian Simulation
+=================================
+
+Shadow Hamiltonian simulation is a new approach to quantum simulation on quantum computers [#SommaShadow]_.
+Despite its name, it has little to do with :doc:`classical shadows </demos/tutorial_diffable_shadows>`.
+In quantum simulation, the goal is typically to simulate the time evolution of expectation values
+of :math:`M` observables :math:`O_m,` for :math:`m=1,\ldots ,M`.
+The common approach is to evolve the wave function :math:`|\psi\rangle` and then measure the desired observables after the evolution.
+
+In shadow Hamiltonian simulation, we instead directly encode the expectation values in a proxy state — the **shadow state** — 
+and evolve that state accordingly. Specifically for time evolution, we can write a shadow Schrödinger equation that governs the
+dynamics of the shadow state.
+
+.. figure:: ../_static/demo_thumbnails/opengraph_demo_thumbnails/OGthumbnail_shadow_hamiltonian_simulation.png
+    :align: center
+    :width: 35%
```

Comment:
```suggestion
    :width: 70%
```
That was the number that was okayed in a Tarik review for OG images.

Reply:
just felt unnecessarily huge here, would you prefer to change it?